### PR TITLE
chore(menu-items): Separate profile and NFT menu items

### PR DESCRIPTION
**Pull request by ChefHutch**
Screenshot 2021-09-27 at 11 20 19
Screenshot 2021-09-27 at 11 20 07

### CODE REVIEW

File: src/components/Menu/UserMenu/ProfileUserMenutItem.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const ProfileUserMenuItem: React.FC<ProfileUserMenuItemProps> = ({ isLoading, ha
   const { t } = useTranslation()
 
   const handleClick = () => {
-    history.push(`${nftsBaseUrl}/profile`)
+    history.push(`${nftsBaseUrl}/profile/achievements`)
```
Comment (Chef-Cheems) [inline src/components/Menu/UserMenu/ProfileUserMenutItem.tsx:24]: why achievements?

**Review by 0xCorgi — APPROVED** [review]

File: src/components/Menu/UserMenu/ProfileUserMenutItem.tsx (hunk 1 above)
Comment (Chef-Cheems) [reply src/components/Menu/UserMenu/ProfileUserMenutItem.tsx:24]: ah, I get it...

**Review by Chef-Cheems — APPROVED** [review]

File: src/components/Menu/UserMenu/ProfileUserMenutItem.tsx (hunk 1 above)
Comment (ChefHutch) [reply src/components/Menu/UserMenu/ProfileUserMenutItem.tsx:24]: just 'cos

Comment (ChefHutch) [reply src/components/Menu/UserMenu/ProfileUserMenutItem.tsx:24]: 'cos designs, 'cos bugs

**Pull request merged**